apps/server/src/modules/account/mapper/account-entity-to-dto.mapper.spec.ts

```diff
@@ -14,101 +14,80 @@ describe('AccountEntityToDtoMapper', () => {
 	});
 
 	describe('mapToDto', () => {
-		it('should map all fields', () => {
-			const testEntity: Account = {
-				_id: new ObjectId(),
-				id: 'id',
-				createdAt: new Date(),
-				updatedAt: new Date(),
-				userId: new ObjectId(),
-				username: 'username',
-				activated: true,
-				credentialHash: 'credentialHash',
-				expiresAt: new Date(),
-				lasttriedFailedLogin: new Date(),
-				password: 'password',
-				systemId: new ObjectId(),
-				token: 'token',
-			};
-			const ret = AccountEntityToDtoMapper.mapToDto(testEntity);
-
-			expect(ret.id).toBe(testEntity.id);
-			expect(ret.createdAt).toEqual(testEntity.createdAt);
-			expect(ret.updatedAt).toEqual(testEntity.createdAt);
-			expect(ret.userId).toBe(testEntity.userId?.toString());
-			expect(ret.username).toBe(testEntity.username);
-			expect(ret.activated).toBe(testEntity.activated);
-			expect(ret.credentialHash).toBe(testEntity.credentialHash);
-			expect(ret.expiresAt).toBe(testEntity.expiresAt);
-			expect(ret.lasttriedFailedLogin).toBe(testEntity.lasttriedFailedLogin);
-			expect(ret.password).toBe(testEntity.password);
-			expect(ret.systemId).toBe(testEntity.systemId?.toString());
-			expect(ret.token).toBe(testEntity.token);
-		});
+		describe('When mapping AccountEntity to AccountDto', () => {
+			const setup = () => {
+				const fullEntity = accountFactory.withAllProperties().buildWithId({}, '000000000000000000000001');
```

Comment:
choose another name for this variable. maybe accountEntity instead of fullEntity